FrontEnd/src/views/kidsCheckUp.vue

```diff
@@ -1,6 +1,6 @@
 <template>
-  <div class="full-div container" style="width: 1280px">
-    <div id="x-contest" class="container-fluid p-3">
+<main class="container-full">
```

Comment:
will these changes work in small mobile view correctly?

Reply:
When I tested in my machine without these changes it looks fine for me. So you can revert the css changes in this file